tools/autograd/gen_autograd.py

```diff
@@ -128,6 +128,8 @@ def load_aten_declarations(path):
 
         for arg in declaration['arguments']:
             arg['simple_type'] = get_simple_type(arg)
+        for arg in declaration['schema_order_arguments']:
+            arg['simple_type'] = get_simple_type(arg)
```

Comment:
What's going on here?

Reply:
`simple_type` is needed downstream for a few things and previously, only the args in `arguments` got it attached, now I'm also attaching it to the args listed in `schema_order_arguments`